projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad-task-definitions.php

```diff
@@ -855,6 +883,30 @@ function wpcom_launchpad_get_newsletter_subscriber_count() {
 	return (int) $total_subscribers;
 }
 
+/**
+ * Determines if Stripe has been connected.
+ *
+ * @return bool Whether or not Stripe account is connected.
+ */
+function wpcom_is_stripe_connected() {
+	require_lib( 'memberships' );
+	$blog_id  = get_current_blog_id();
+	$settings = (array) get_memberships_settings_for_site( $blog_id );
+	return boolval( $settings['connected_account_id'] );
+}
+
+/**
+ * Determines if paid membership plans exists
+ *
+ * @return bool Whether or not paid membership plans exist.
+ */
+function wpcom_has_paid_membership_plans() {
```

Comment:
As above RE naming, can we please keep the `wpcom_launchpad_` prefix, or move the code elsewhere?
```suggestion
function wpcom_launchpad_has_paid_membership_plans() {
```

Reply:
Sure! Happy to update.

Reply:
Updated method names.